src/dash.ts

```diff
@@ -33,4 +34,35 @@ export class Dash {
 
         return '';
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns the command to handle the URI in the command line.
+     *
+     * @param {string} OS - string representing the OS
+     * @return {string} name and flags of the command that takes the URI as an argument
+     */
+    getDashURIHandler(OS: string): string {
+        // hacky switch statement
+        return {
+            'darwin': 'open -g',
+            'linux': 'zeal',
+            // start Zeal before we pass it a URI so it works whether Zeal is running in background or not.
+            // Why is this needed? I have no idea, but I think Qt has to start up a listener on cmd before it will work.
+            'win32': 'start dash-plugin:// && start'
+        }[OS] || 'zeal';
+    }
+
+    /**
+     * Returns the OS specific URI to be handled.
+     *
+     * @param {string} uri - original constructed URI
+     * @param {string} OS - string representing the OS
+     * @return {string} OS-specific URI to pass to handler, mainly because of Windows
+     */
+    getOSSpecificURI(uri: string, OS: string): string {
+        return {
+            // on Windows, start can't deal with the double quotes, and & needs to be escaped with ^
+            'win32': uri.replace('&', '^&')
+        }[OS] || '"' + uri + '"';
+    }
```

Comment:
there is another approach for specifying OS. Instead of defining it as parameter in every method, we can just put in the constructor of this Dash class. 

```typescript
 export class Dash {
   private OS: string;

   constructor(OS: string) {
       this.OS = OS;
   }
}
```
The benefits of this way is: 
1) only assign OS one time in the constructor
2) method won't need extra param, they can access the OS by using `this.OS`

What do you think?